### DISCUSSION TITLE
How to limit answer to be sourced only from documents?

### DISCUSSION THREAD

**Discussion by johnkmayo** [PromtEngineer/localGPT#106] [Q&A]
I'm finally gotten this up and running locally. Unfortunately when I asked a question that I was sure couldn't be answered from the constitution.pdf file, I still got an answer. Is it possible to limit the response to be based **_only_** on the contents of the documents?

**Comment by johnkmayo** [comment]
Prefixing the question with "Using only the sources documents provided, " seems to help but I'm interested in how other people are handling this.

**Comment by PromtEngineer** [comment]
I am working on a prompt in the PromptTemplate with the help of which we will be able to (in most cases) do that.
